docker/phpstan/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# phpstan in docker
+# used as sanity check of code when installed via "composer install --no-dev" (so on prod) w/o other dev tools to check
+
+FROM phpstan/phpstan:latest
+
+RUN echo "go" \
```

Comment:
You already explained that this was common practice to have the double ampersands in the lines below aligned. I'm not opposing this and there is a lot of output produced anyway. But what do you think about printing something meaningful rather than just "go"?